<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:st1="urn:schemas-microsoft-com:office:smarttags"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 11">
<meta name=Originator content="Microsoft Word 11">
<link rel=File-List href="suppl_files/filelist.xml">
<o:SmartTagType namespaceuri="urn:schemas-microsoft-com:office:smarttags"
 name="country-region"/>
<o:SmartTagType namespaceuri="urn:schemas-microsoft-com:office:smarttags"
 name="City"/>
<o:SmartTagType namespaceuri="urn:schemas-microsoft-com:office:smarttags"
 name="PlaceType"/>
<o:SmartTagType namespaceuri="urn:schemas-microsoft-com:office:smarttags"
 name="PlaceName"/>
<o:SmartTagType namespaceuri="urn:schemas-microsoft-com:office:smarttags"
 name="place"/>
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>nanjiang</o:Author>
  <o:LastAuthor>nanjiang</o:LastAuthor>
  <o:Revision>99</o:Revision>
  <o:TotalTime>146</o:TotalTime>
  <o:Created>2009-08-06T14:56:00Z</o:Created>
  <o:LastSaved>2009-08-11T12:02:00Z</o:LastSaved>
  <o:Pages>1</o:Pages>
  <o:Words>832</o:Words>
  <o:Characters>4415</o:Characters>
  <o:Company>stockholm university</o:Company>
  <o:Lines>36</o:Lines>
  <o:Paragraphs>10</o:Paragraphs>
  <o:CharactersWithSpaces>5237</o:CharactersWithSpaces>
  <o:Version>11.9999</o:Version>
 </o:DocumentProperties>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:Zoom>90</w:Zoom>
  <w:DontDisplayPageBoundaries/>
  <w:HyphenationZone>21</w:HyphenationZone>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:UseFELayout/>
  </w:Compatibility>
  <w:BrowserLevel>MicrosoftInternetExplorer4</w:BrowserLevel>
 </w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" LatentStyleCount="156">
 </w:LatentStyles>
</xml><![endif]--><!--[if !mso]><object
 classid="clsid:38481807-CA0E-42D2-BF39-B33AF135CC4D" id=ieooui></object>
<style>
st1\:*{behavior:url(#ieooui) }
</style>
<![endif]-->
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;
	mso-font-charset:2;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:0 268435456 0 0 -2147483648 0;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;
	mso-font-alt:\5B8B\4F53;
	mso-font-charset:134;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:3 135135232 16 0 262145 0;}
@font-face
	{font-family:"\@SimSun";
	panose-1:2 1 6 0 3 1 1 1 1 1;
	mso-font-charset:134;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:3 135135232 16 0 262145 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-parent:"";
	margin:0cm;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:SimSun;
	mso-ansi-language:EN-GB;}
h3
	{mso-style-next:Normal;
	margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:0cm;
	mso-pagination:widow-orphan;
	page-break-after:avoid;
	mso-outline-level:3;
	font-size:13.0pt;
	font-family:Arial;
	mso-ansi-language:EN-GB;}
a:link, span.MsoHyperlink
	{color:blue;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{color:purple;
	text-decoration:underline;
	text-underline:single;}
span.moz-txt-tag
	{mso-style-name:moz-txt-tag;}
@page Section1
	{size:595.3pt 841.9pt;
	margin:70.85pt 70.85pt 70.85pt 70.85pt;
	mso-header-margin:35.4pt;
	mso-footer-margin:35.4pt;
	mso-paper-source:0;}
div.Section1
	{page:Section1;}
 /* List Definitions */
 @list l0
	{mso-list-id:987170819;
	mso-list-type:hybrid;
	mso-list-template-ids:402582046 69009433 69009433 69009435 69009423 69009433 69009435 69009423 69009433 69009435;}
@list l0:level1
	{mso-level-number-format:alpha-lower;
	mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	margin-left:108.0pt;
	text-indent:-18.0pt;}
@list l0:level2
	{mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level3
	{mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level4
	{mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level5
	{mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level6
	{mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level7
	{mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level8
	{mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level9
	{mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1
	{mso-list-id:1229460443;
	mso-list-type:hybrid;
	mso-list-template-ids:-135634718 69009417 69009425 69009413 69009409 69009411 69009413 69009409 69009411 69009413;}
@list l1:level1
	{mso-level-number-format:bullet;
	mso-level-text:\F076;
	mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	font-family:Wingdings;}
@list l1:level2
	{mso-level-text:"%2\)";
	mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level3
	{mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level4
	{mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level5
	{mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level6
	{mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level7
	{mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level8
	{mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l1:level9
	{mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2
	{mso-list-id:1690907907;
	mso-list-type:hybrid;
	mso-list-template-ids:106717994 69009433 69009433 69009435 69009423 69009433 69009435 69009423 69009433 69009435;}
@list l2:level1
	{mso-level-number-format:alpha-lower;
	mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	margin-left:108.0pt;
	text-indent:-18.0pt;}
@list l2:level2
	{mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level3
	{mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level4
	{mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level5
	{mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level6
	{mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level7
	{mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level8
	{mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l2:level9
	{mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3
	{mso-list-id:1756710050;
	mso-list-type:hybrid;
	mso-list-template-ids:1690570340 69009433 69009433 69009435 69009423 69009433 69009435 69009423 69009433 69009435;}
@list l3:level1
	{mso-level-number-format:alpha-lower;
	mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	margin-left:108.0pt;
	text-indent:-18.0pt;}
@list l3:level2
	{mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level3
	{mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level4
	{mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level5
	{mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level6
	{mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level7
	{mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level8
	{mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l3:level9
	{mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4
	{mso-list-id:2007198450;
	mso-list-type:hybrid;
	mso-list-template-ids:-1390101780 69009423 69009425 69009435 69009423 69009433 69009435 69009423 69009433 69009435;}
@list l4:level1
	{mso-level-tab-stop:36.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level2
	{mso-level-text:"%2\)";
	mso-level-tab-stop:72.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level3
	{mso-level-tab-stop:108.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level4
	{mso-level-tab-stop:144.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level5
	{mso-level-tab-stop:180.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level6
	{mso-level-tab-stop:216.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level7
	{mso-level-tab-stop:252.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level8
	{mso-level-tab-stop:288.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l4:level9
	{mso-level-tab-stop:324.0pt;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";
	mso-ansi-language:#0400;
	mso-fareast-language:#0400;
	mso-bidi-language:#0400;}
</style>
<![endif]-->
</head>

<body lang=SV link=blue vlink=purple style='tab-interval:36.0pt'>

<div class=Section1>

<h3><span lang=EN-GB>Supplementary data for the paper “A novel method for accurate
one-dimensional </span></h3>

<h3><span lang=EN-GB>protein structure prediction based on fragment matching”</span></h3>

<p class=MsoNormal><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-GB>Authors: Tuping Zhou, Nanjiang Shu and Sven
Hovmöller</span></p>

<p class=MsoNormal><span lang=EN-GB>Structural Chemistry, <st1:PlaceName w:st="on">Stockholm</st1:PlaceName>
<st1:PlaceType w:st="on">University</st1:PlaceType>, SE-106 91, <st1:place
w:st="on"><st1:City w:st="on">Stockholm</st1:City>, <st1:country-region w:st="on">Sweden</st1:country-region></st1:place></span></p>

<p class=MsoNormal><span lang=EN-GB>Contact: Nanjiang Shu <a
href="mailto:nanjiang@struc.su.se">nanjiang@struc.su.se</a></span></p>

<p class=MsoNormal><span lang=EN-GB>Updated 2009-08-11</span></p>

<p class=MsoNormal><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><b style='mso-bidi-font-weight:normal'><span lang=EN-GB>Abbreviations:<o:p></o:p></span></b></p>

<p class=MsoNormal><span lang=EN-GB>PDB: Protein Data Bank</span></p>

<p class=MsoNormal><span lang=EN-GB>Q3:<span style='mso-spacerun:yes'>  
</span>overall per-residue accuracy for three-state secondary structure
prediction</span></p>

<p class=MsoNormal><span lang=EN-GB>S3:<span style='mso-spacerun:yes'>   
</span>overall per-residue accuracy for three-state Shape String prediction</span></p>

<p class=MsoNormal><span lang=EN-GB>S8:<span style='mso-spacerun:yes'>   
</span>overall per-residue accuracy for eight-state Shape String prediction</span></p>

<p class=MsoNormal><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<ol style='margin-top:0cm' start=1 type=1>
 <li class=MsoNormal style='mso-list:l4 level1 lfo1;tab-stops:list 36.0pt'><span
     lang=EN-GB>The list of PDB chains and amino acid sequences in FASTA format
     for three datasets cutting at &#8804;30%, &#8804;25% and &#8804;20%
     sequence identity, respectively.</span></li>
</ol>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l4 level2 lfo1;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>1)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>Cutting at &#8804;30% sequence
identity, including 5860 chains: <a href="nr30.idlist">chain id list</a>, <a
href="nr30.fasta">amino acid sequence file</a></span></p>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l4 level2 lfo1;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>2)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>Cutting at &#8804;25% sequence
identity, including 4227 chains: <a href="nr25.idlist">chain id list</a>, <a
href="nr25.fasta">amino acid sequence file</a></span></p>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l4 level2 lfo1;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>3)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>Cutting at &#8804;25% sequence
identity, including 3338 chains: <a href="nr20.idlist">chain id list</a>, <a
href="nr20.fasta">amino acid sequence file</a></span></p>

<p class=MsoNormal><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<ol style='margin-top:0cm' start=2 type=1>
 <li class=MsoNormal style='mso-list:l4 level1 lfo1;tab-stops:list 36.0pt'><span
     lang=EN-GB>The training set and test set for benchmarking with PSIPRED
     version 2.61</span></li>
</ol>

<p class=MsoNormal><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:36.0pt'><span lang=EN-GB>We thank Dr.
David Jones for providing us the training set, which was used to build the
weight files for PSIPRED version 2.61, for carrying out the benchmark.<br>
The training set contains 6598 protein chains, 1563587 amino acids, with the
average sequence length =237 amino acids.<br>
The chain id is <a href="psipred2.61.training.idlist">psipred2.61.training.idlist</a>
and the sequence file is <a href="psipred2.61.training.fasta">psipred2.61.training.fasta</a>.<br>
<br>
Note that many sequences in this training set are of high sequence identity to
each other. For example, the sequence identity of the chain 1JPTL and 1L7IL is
as high as 90%. </span></p>

<p class=MsoNormal style='margin-left:36.0pt'><span lang=EN-GB>When cutting
this training set down to &#8804;30% sequence identity, only 3644 chains
remain. The chain id list for this non redundant set of the training set can be
found <br>
at <a href="psipred2.61.training.nr30.idlist">psipred2.61.training.nr30.idlist</a>.<br>
<br>
The test set was constructed in the following ways. </span></p>

<p class=MsoNormal style='margin-left:36.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l1 level2 lfo2;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>1)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>Step 1: Obtain all PDB chains
(as of June 10, 2009) cutting at &#8804;99% sequence identity, with resolution
&lt;2.5Å and R-value &lt;0.3 and using only X-ray structures. <br>
This returned 21574 protein chains. </span></p>

<p class=MsoNormal style='margin-left:54.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l1 level2 lfo2;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>2)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>Step 2: Out of thus obtained
21574 chains, those chains with the same chain id with any one in the training
set were removed. <br>
This resulted in 15256 protein chains.</span></p>

<p class=MsoNormal style='margin-left:36.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l1 level2 lfo2;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>3)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>Step 3: Run blastpgp for all
these15256 chains against the training set and those chains with at least one
significant hit from the training set were removed. <br>
The parameter for blastpgp is: E-value threshold = 1e-3 AND Iteration = 3.<br>
The criteria for being a significant hit are: (Sequence identity &gt;30% AND
Alignment length &gt;30 AND E-value &lt;0.1) <br>
<span
style='mso-spacerun:yes'>                                                      
</span>OR (Sequence identity &gt;50% AND Alignment length &gt;15 AND E-value
&lt;5) <br>
This resulted in 3100 protein chains.</span></p>

<p class=MsoNormal style='margin-left:54.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l1 level2 lfo2;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>4)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>Step 4: Cut these 3100 protein
chains down to &#8804;30% sequence identity.<br>
This resulted in 2421 protein chains.</span></p>

<p class=MsoNormal style='margin-left:54.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:18.0pt;text-indent:18.0pt'><span
lang=EN-GB>The chain id list of the test set can be found at <a
href="test2421.idlist">test2421.idlist</a> and the amino acid sequence file can
be found at <a href="test2421.fasta">test2421.fasta</a>. </span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='text-indent:36.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<ol style='margin-top:0cm' start=3 type=1>
 <li class=MsoNormal style='mso-list:l4 level1 lfo1;tab-stops:list 36.0pt'><span
     lang=EN-GB>Prediction results on protein chains cutting at &#8804;30%
     (5860 chains), &#8804;25% (4227 chains) and &#8804;20% (3338 chains)
     sequence identity, respectively.</span></li>
</ol>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l4 level2 lfo1;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>1)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>For the dataset cutting at &#8804;30%
sequence identity, including 5860 chains</span></p>

<p class=MsoNormal style='margin-left:108.0pt;text-indent:-18.0pt;mso-list:
l2 level1 lfo3;tab-stops:list 108.0pt'><![if !supportLists]><span lang=EN-GB
style='mso-fareast-font-family:"Times New Roman"'><span style='mso-list:Ignore'>a.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-GB>Q3 for each chain and the overall Q3 can be found at <a
href="nr30.Q3.list">nr30.Q3.list</a></span></p>

<p class=MsoNormal style='margin-left:108.0pt;text-indent:-18.0pt;mso-list:
l2 level1 lfo3;tab-stops:list 108.0pt'><![if !supportLists]><span lang=EN-GB
style='mso-fareast-font-family:"Times New Roman"'><span style='mso-list:Ignore'>b.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-GB>S3 and S8 for each chain and the overall S3 and S8 can be found at <a
href="nr30.S3S8.list">nr30.S3S8.list</a></span></p>

<p class=MsoNormal style='margin-left:72.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l4 level2 lfo1;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>2)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>For the dataset cutting at &#8804;25%
sequence identity, including 4227 chains</span></p>

<p class=MsoNormal style='margin-left:108.0pt;text-indent:-18.0pt;mso-list:
l0 level1 lfo4;tab-stops:list 108.0pt'><![if !supportLists]><span lang=EN-GB
style='mso-fareast-font-family:"Times New Roman"'><span style='mso-list:Ignore'>a.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-GB>Q3 for each chain and the overall Q3 can be found at <a
href="nr25.Q3.list">nr25.Q3.list</a></span></p>

<p class=MsoNormal style='margin-left:108.0pt;text-indent:-18.0pt;mso-list:
l0 level1 lfo4;tab-stops:list 108.0pt'><![if !supportLists]><span lang=EN-GB
style='mso-fareast-font-family:"Times New Roman"'><span style='mso-list:Ignore'>b.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-GB>S3 and S8 for each chain and the overall S3 and S8 can be found at <a
href="nr25.S3S8.list">nr25.S3S8.list</a></span></p>

<p class=MsoNormal style='margin-left:90.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l4 level2 lfo1;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>3)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>For the dataset cutting at &#8804;20%
sequence identity, including 3338 chains</span></p>

<p class=MsoNormal style='margin-left:108.0pt;text-indent:-18.0pt;mso-list:
l3 level1 lfo5;tab-stops:list 108.0pt'><![if !supportLists]><span lang=EN-GB
style='mso-fareast-font-family:"Times New Roman"'><span style='mso-list:Ignore'>a.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-GB>Q3 for each chain and the overall Q3 can be found at <a
href="nr20.Q3.list">nr20.Q3.list</a></span></p>

<p class=MsoNormal style='margin-left:108.0pt;text-indent:-18.0pt;mso-list:
l3 level1 lfo5;tab-stops:list 108.0pt'><![if !supportLists]><span lang=EN-GB
style='mso-fareast-font-family:"Times New Roman"'><span style='mso-list:Ignore'>b.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-GB>S3 and S8 for each chain and the overall S3 and S8 can be found at <a
href="nr20.S3S8.list">nr20.S3S8.list</a></span></p>

<p class=MsoNormal style='margin-left:36.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:18.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:18.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<ol style='margin-top:0cm' start=4 type=1>
 <li class=MsoNormal style='mso-list:l4 level1 lfo1;tab-stops:list 36.0pt'><span
     lang=EN-GB>Prediction results for benchmarking with PSIPRED2.61</span></li>
</ol>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l4 level2 lfo1;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>1)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>Prediction results by our
method Frag1D</span></p>

<p class=MsoNormal style='margin-left:54.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:108.0pt;text-indent:-18.0pt;mso-list:
l4 level3 lfo1;tab-stops:list 108.0pt'><![if !supportLists]><span lang=EN-GB
style='mso-fareast-font-family:"Times New Roman"'><span style='mso-list:Ignore'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-GB>Q3 for each chain and the overall Q3 can be found at <a
href="benchFrag1D.Q3.list">benchFrag1D.Q3.list</a></span></p>

<p class=MsoNormal style='margin-left:90.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l4 level2 lfo1;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>2)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>Prediction results by PSIPRED2.61</span></p>

<p class=MsoNormal style='margin-left:108.0pt;text-indent:-18.0pt;mso-list:
l4 level3 lfo1;tab-stops:list 108.0pt'><![if !supportLists]><span lang=EN-GB
style='mso-fareast-font-family:"Times New Roman"'><span style='mso-list:Ignore'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-GB>Q3 for each chain and the overall Q3 can be found at <a
href="benchPSIPRED261.Q3.list">benchPSIPRED261.Q3.list</a></span></p>

<p class=MsoNormal style='margin-left:36.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<ol style='margin-top:0cm' start=5 type=1>
 <li class=MsoNormal style='mso-list:l4 level1 lfo1;tab-stops:list 36.0pt'><span
     lang=EN-GB>Prediction results on the dataset containing 1296 chains which
     has been used by Kuang et al. (2004) for Shape String prediction</span></li>
</ol>

<p class=MsoNormal style='margin-left:72.0pt;text-indent:-18.0pt;mso-list:l4 level2 lfo1;
tab-stops:list 72.0pt'><![if !supportLists]><span lang=EN-GB style='mso-fareast-font-family:
"Times New Roman"'><span style='mso-list:Ignore'>1)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]><span lang=EN-GB>S3 and S8 for each chain and
the overall S3 and S8 can be found at <a href="db1296.S3S8.list">db1296.S3S8.list</a></span></p>

<p class=MsoNormal style='margin-left:54.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:54.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<ol style='margin-top:0cm' start=6 type=1>
 <li class=MsoNormal style='line-height:200%;mso-list:l4 level1 lfo1;
     tab-stops:list 36.0pt'><span lang=EN-GB>The lists of 50 randomly selected
     PDB chains and their corresponding templates for homology modelling can be
     found at <a href="modellerRand50.list">modellerRand50.list</a></span></li>
</ol>

<p class=MsoNormal style='margin-left:18.0pt;line-height:200%'><span
lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:36.0pt;line-height:200%'><span
lang=EN-GB><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-left:18.0pt'><span lang=EN-GB><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
